src/fs/rock/RockRebuild.h

```diff
@@ -52,6 +52,9 @@ class Rebuild: public AsyncJob, private IndependentRunner
     /// \returns whether the indexing was necessary (and, hence, started)
     static bool Start(SwapDir &dir);
 
+    /* AsyncJob API */
+    virtual void callException(const std::exception &) override;
```

Comment:
Please make this method `protected` if possible.

Reply:
It looks like this change request was missed. Please note that, if making this method protected is possible, then it should go inside the existing "AsyncJob API" group in the `protected` section of this class declaration.

Reply:
No, this does not work:
```
../../src/base/AsyncJobCalls.h:184:9: error: ‘virtual void Rock::Rebuild::callException(const std::exception&)’ is protected within this context
         job->callException(e);
```

Reply:
Understood. This can probably be fixed/improved, but those improvements are out of this PR scope. Resolving.